src/popover.ts

```diff
@@ -214,13 +219,17 @@ export function apply() {
     togglePopover: {
       enumerable: true,
       configurable: true,
-      value(force: boolean) {
+      value(options: boolean | PopoverTogglePopoverOptions = {}) {
```

Comment:
While reviewing this, I realized that `togglePopover` is now supposed to return a boolean indicating whether the popover is in the "showing" state. I think we have the tools to do that -- do you want to either add that here or file a new issue?

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/togglePopover#return_value

Reply:
Good catch. I'm going to file a new issue, because I uncovered a minor spec deviation in the case of `Popover is not showing, and force = false`. Currently we are running `hidePopover`, but we should only be running `checkPopoverValidity`. I want to make sure there aren't significant issues there, and it may change what we'd return slightly.